Gopkg.lock

```diff
@@ -1666,10 +1666,12 @@
     "k8s.io/client-go/discovery",
     "k8s.io/client-go/dynamic",
     "k8s.io/client-go/informers",
+    "k8s.io/client-go/informers/autoscaling/v2beta1",
```

Comment:
Should this be included ?

Reply:
These are packages we are using for features in the DCA, so yes I think they do need to be in here.